subliminal/providers/itasa.py

```diff
@@ -0,0 +1,390 @@
+        self.logged_in = False
+
+    @region.cache_on_arguments(expiration_time=SHOW_EXPIRATION_TIME)
+    def _get_show_ids(self):
+        """Get the ``dict`` of show ids per series by querying the `shows` page.
+
+        :return: show id per series, lower case and without quotes.
+        :rtype: dict
+
+        """
+        # get the show page
+        logger.info('Getting show ids')
+        params = {'apikey': self.apikey}
+        r = self.session.get(self.server_url + 'shows', timeout=10, params=params)
+        r.raise_for_status()
+        root = ParserBeautifulSoup(r.content, ['lxml-xml', 'lxml', 'html.parser']).find('root')
+
+        # populate the show ids
+        show_ids = {}
+        for show in root.shows:
+            show_ids[sanitize_string(show.find('name').text).lower()] = int(show.id.text)
+        logger.debug('Found %d show ids', len(show_ids))
+
+        return show_ids
+
+    @region.cache_on_arguments(expiration_time=SHOW_EXPIRATION_TIME)
+    def _search_show_id(self, series):
+        """Search the show id from the `series`
+
+        :param str series: series of the episode.
+        :return: the show id, if found.
+        :rtype: int or None
+
+        """
+        # build the param
+        params = {'apikey': self.apikey, 'q': series}
+
+        # make the search
+        logger.info('Searching show ids with %r', params)
+        r = self.session.get(self.server_url + 'shows/search', params=params, timeout=10)
+        r.raise_for_status()
+        root = ParserBeautifulSoup(r.content, ['lxml-xml', 'lxml', 'html.parser']).find('root')
+
+        if int(root.data.count.text) == 0:
+            logger.warning('Show id not found: no suggestion')
+            return None
+
+        # Looking for show in first page
+        for show in root.data.shows:
+            if sanitized_string_equal(show.find('name').text.lower(), series.lower()):
+                show_id = int(show.id.text)
+                logger.debug('Found show id %d', show_id)
+
+                return show_id
+
+        # Not in the first page of result try next (if any)
+        next = root.find('next')
+        while next.text != '':
+
+            r = self.session.get(next.text, timeout=10)
+            r.raise_for_status()
+            root = ParserBeautifulSoup(r.content, ['lxml-xml', 'lxml', 'html.parser']).find('root')
+
+            logger.info('Loading suggestion page %s', root.data.page.text)
+
+            # Looking for show in following pages
+            for show in root.data.shows:
+                if sanitized_string_equal(show.find('name').text.lower(), series.lower()):
+                    show_id = int(show.id)
+                    logger.debug('Found show id %d', show_id)
+
+                    return show_id
+
+            next = root.find('next')
+
+        # No matches found
+        logger.warning('Show id not found: suggestions does not match')
+
+        return None
+
+    def get_show_id(self, series, country_code=None):
```

Comment:
As I saw on addic7ed provider, from which i copied, same show can have local variation such  UK, US. 
This fit that cases.